packages/cli/src/environments.ee/source-control/source-control.service.ee.ts

```diff
@@ -512,10 +518,10 @@ export class SourceControlService {
 
 		// #region Tracking Information
 		if (options.direction === 'push') {
-			this.eventService.emit(
-				'source-control-user-started-push-ui',
-				getTrackingInformationFromPrePushResult(sourceControlledFiles),
-			);
+			this.eventService.emit('source-control-user-started-push-ui', {
+				...getTrackingInformationFromPrePushResult(sourceControlledFiles),
+				userId: user?.id,
+			});
 		} else if (options.direction === 'pull') {
 			this.eventService.emit(
 				'source-control-user-started-pull-ui',
```

Comment:
Is it worth doing this for the `pull` events too? They'll have the same issue. Or do this as a separate fix?